lib/ansible/modules/cloud/docker/docker_compose.py

```diff
@@ -50,7 +50,8 @@
   state:
     description:
       - Desired state of the project.
-      - Specifying I(present) is the same as running I(docker-compose up).
+      - Specifying I(present) is the same as running I(docker-compose up) / I(docker-compose stop) (with C(stopped)) / I(docker-compose restart)
```

Comment:
```suggestion
      - Specifying C(present) is the same as running C(docker-compose up) / C(docker-compose stop) (with I(stopped)) / C(docker-compose restart)
```
`I(...)` is used for Ansible option names, `C(...)` is used for option values and other stuff. (Also applies to many other changes.)

Reply:
Also, I find this notation with `/` hard to read. I would rather write `resp.` instead of `/`.

Reply:
Oh ok. Then it was/is "wrong" in the whole module. I will change that.